common3.json

```diff
@@ -5355,7 +5355,7 @@
     },
     {
       "id" : 61343,
-      "text" : "ヘボイ"
+      "text" : "Folly "
```

Comment:
trailing space (don`t fix btw, this is just a comment to remind myself what to do since I will be merging this)